### PR TITLE
Add arm swing event

### PR DESCRIPTION
**Pull request by Romitou**
### Description
This simple PR adds an event called when a player swings his arm.
It does not provide a comparison of the type of animation, as there is only one type (`ARM_SWING`).

---
**Target Minecraft Versions:** any
**Requirements:** none
**Related Issues:** #3114

### CODE REVIEW

File: src/main/java/ch/njol/skript/events/SimpleEvents.java [hunk 1]
```diff
@@ -571,12 +572,17 @@
 			.description("Called when a horse jumps.")
 			.examples("on horse jump:", "\tpush event-entity upwards at speed 2")
 			.since("INSERT VERSION");
-		if(Skript.classExists("org.bukkit.event.block.BlockFertilizeEvent"))
+		if (Skript.classExists("org.bukkit.event.block.BlockFertilizeEvent"))
```
Comment (ShaneBeee) [inline src/main/java/ch/njol/skript/events/SimpleEvents.java:575]: While I understand this space needs to be here, this should not be part of this PR. Please revery this change.
Changes should only be done which are in regards to the issue/change at hand.

File: src/main/java/ch/njol/skript/events/SimpleEvents.java [hunk 2]
```diff
@@ -571,12 +572,17 @@
...
 			Skript.registerEvent("Block Fertilize", SimpleEvent.class, BlockFertilizeEvent.class, "[block] fertilize")
 			.description("Called when a player fertilizes blocks.")
 			.requiredPlugins("Minecraft 1.13 or newer")
 			.examples("on block fertilize:",
 				"\tsend \"Fertilized %size of fertilized blocks% blocks got fertilized.\"")
 			.since("2.5");
+		Skript.registerEvent("Arm Swing", SimpleEvent.class, PlayerAnimationEvent.class, "[player] [arm] swing")
```
Comment (ShaneBeee) [inline src/main/java/ch/njol/skript/events/SimpleEvents.java:582]: in my opinion, "arm" should not be optional.
"on swing" is pretty vague.
In the event that in the future something else regarding "swing" is added, it could cause issues.

**Review by ShaneBeee — CHANGES_REQUESTED** [review]
some small changes

File: src/main/java/ch/njol/skript/events/SimpleEvents.java (hunk 1 above)
Comment (Romitou) [reply src/main/java/ch/njol/skript/events/SimpleEvents.java:575]: Oops, you're right. Will be reverted.

**Review by ShaneBeee — APPROVED** [review]
I changed the branch from master -> dev-2.5
It created a conflict but I fixed that :)

Looks good to me

**Review by APickledWalrus — APPROVED** [review]

**Comment by APickledWalrus** [comment]
Thank you for your contribution
